WalletWasabi.Gui/Controls/WalletExplorer/SendTabViewModel.cs

```diff
@@ -686,37 +690,42 @@ private void SetFeesAndTexts()
 				}
 				else
 				{
-					switch (FeeDisplayFormat)
-					{
-						case FeeDisplayFormat.SatoshiPerByte:
-							FeeText = $"(~ {FeeRate.SatoshiPerByte} sat/byte)";
-							FeeToolTip = "Expected fee rate in satoshi / vbyte.";
-							break;
-
-						case FeeDisplayFormat.USD:
-							FeeText = $"(~ ${UsdFee.ToString("0.##")})";
-							FeeToolTip = $"Estimated total fees in USD. Exchange Rate: {(long)UsdExchangeRate} BTC/USD.";
-							break;
-
-						case FeeDisplayFormat.BTC:
-							FeeText = $"(~ {EstimatedBtcFee.ToString(false, false)} BTC)";
-							FeeToolTip = "Estimated total fees in BTC.";
-							break;
-
-						case FeeDisplayFormat.Percentage:
-							FeeText = $"(~ {FeePercentage.ToString("0.#")} %)";
-							FeeToolTip = "Expected percentage of fees against the amount to be sent.";
-							break;
-
-						default:
-							throw new NotSupportedException("This is impossible.");
-					}
+					SetFeeDisplay();
```

Comment:
`SetFeesAndTexts();` is a bad practice. We should not double down on this for the sake of this PR.

Reply:
How do you want me to do this? Do you want me to do something like:

```c#
public string FeeText => FeeDisplayFormat == FeeDisplayFormat.SatoshiPerByte ? $"(~{FeeRate.SatoshiPerByte} sat/byte)" :  /* the rest of them */;
```

Reply:
> How do you want me to do this?

I don't know. Sorry, but I don't have time to rework pull requests anymore like how I kept doing it previously, sorry if it feels like I'm just criticising, which is kind of true, but knowing what not to do is while worse than knowing what to do, it is more valuable than not knowing what not to do.